front/lib/api/invitation.ts

```diff
@@ -25,29 +25,75 @@ function typeFromModel(
   };
 }
 
-export async function createInvitation(
+export async function createOrRecreateInvitation(
```

Comment:
per IRL conversation, call this function `getOrCreaateInvitation()` and do not revoke pending invitations when one already exists.

Reply:
AS per our re-discussion, ok :)